rules/local-development/wp-umbrella-backup-import.md

```diff
@@ -0,0 +1,54 @@
+# WP-Umbrella Backup Import in DDEV
+
+Process for importing WP-Umbrella backups into DDEV local development environments.
+
+## Download and Extract Backup
+
+Move the downloaded `.tar.gz` backup file to your DDEV project folder and extract:
+
+```bash
+tar -xvzf filename.tar.gz
+```
+
+## Import Database
+
+Ensure your project is configured (`ddev config`) and running (`ddev start`). Enter the DDEV container and import all SQL files:
+
+```bash
+ddev ssh
+cd umb_database
+for file in *.sql; do mysql -u db -p'db' db < "$file"; done
+exit
+```
+
+## WordPress Configuration
+
+Edit `wp-config.php` and remove old database credentials (`DB_NAME`, `DB_USER`, `DB_PASSWORD`, `DB_HOST`) and debug settings (`WP_DEBUG`, `WP_ENVIRONMENT_TYPE`) as DDEV manages these.
```

Comment:
This behaviour changed in a recent ddev version. Add WP_ENVIRONMENT_TYPE local to wp-config as this is no longer done automatically by ddev